src/library/Box/types.js

```diff
@@ -7,7 +7,9 @@ type SpacingSize = $Keys<typeof SPACING_SIZE>;
 type InsetSpacingSize = $Keys<typeof INSET_SPACING_SIZE>;
 
 export type BoxProps = {
+  borderRadius?: string | Array<string>,
```

Comment:
list ours up top.

Also, what is this file? Is this Flow?

Reply:
Yeah, this is all of their flow types.